### PR TITLE
Move clickhouse queries of death to public docs

### PR DESCRIPTION
**Pull request by volokluev**

### CODE REVIEW

File: docs/source/clickhouse/death_queries.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Clickhouse Queries Of Death
+===========================
+
+
+The following queries have been shown to segfault clickhouse on 20.7 (which is what snuba is running). Do not run these queries in the tracing tool, unless you really want to take clickhouse down.
```
Comment (onewland) [inline docs/source/clickhouse/death_queries.rst:5]:
```suggestion
The following queries have been shown to segfault clickhouse on 20.7 (which is the minimum Clickhouse version of Snuba). Do not run these queries in the tracing tool, unless you really want to take clickhouse down.
```

for our open source users

**Review by onewland — APPROVED** [review]
Seems fine. We should settle on a consistent way of capitalizing ClickHouse (which looks like how they do it on clickhouse.com)

File: docs/source/clickhouse/death_queries.rst (hunk 1 above)
Comment (onewland) [reply docs/source/clickhouse/death_queries.rst]: Also, did this segfault 20.3? Because I think that's the actual min version in the console

Comment (volokluev) [reply docs/source/clickhouse/death_queries.rst]: I'm pretty sure we run 20.7, but also possibly 20.3?

**Pull request merged**